.rubocop.yml

```diff
@@ -105,6 +105,81 @@ Performance/StringIdentifierArgument: # new in 1.13
 Performance/Sum: # new in 1.8
   Enabled: true
 
+Rails/ActionControllerTestCase: # new in 2.14
+  Enabled: true
+Rails/ActiveRecordCallbacksOrder: # new in 2.7
+  Enabled: true
+Rails/AddColumnIndex: # new in 2.11
+  Enabled: true
+Rails/AfterCommitOverride: # new in 2.8
+  Enabled: true
+Rails/AttributeDefaultBlockValue: # new in 2.9
+  Enabled: true
+Rails/CompactBlank: # new in 2.13
+  Enabled: true
+Rails/DeprecatedActiveModelErrorsMethods: # new in 2.14
+  Enabled: true
+Rails/DuplicateAssociation: # new in 2.14
+  Enabled: true
+Rails/DuplicateScope: # new in 2.14
+  Enabled: true
+Rails/DurationArithmetic: # new in 2.13
+  Enabled: true
+Rails/EagerEvaluationLogMessage: # new in 2.11
+  Enabled: true
+Rails/ExpandedDateRange: # new in 2.11
+  Enabled: true
+Rails/FindById: # new in 2.7
+  Enabled: true
+Rails/I18nLazyLookup: # new in 2.14
+  Enabled: true
+Rails/I18nLocaleAssignment: # new in 2.11
+  Enabled: true
+Rails/I18nLocaleTexts: # new in 2.14
+  Enabled: true
+Rails/Inquiry: # new in 2.7
+  Enabled: true
+Rails/MailerName: # new in 2.7
+  Enabled: true
+Rails/MatchRoute: # new in 2.7
+  Enabled: true
+Rails/MigrationClassName: # new in 2.14
+  Enabled: true
+Rails/NegateInclude: # new in 2.7
+  Enabled: true
+Rails/Pluck: # new in 2.7
+  Enabled: true
+Rails/PluckInWhere: # new in 2.7
+  Enabled: true
+Rails/RedundantPresenceValidationOnBelongsTo: # new in 2.13
+  # Cop gives false positives
+  # Perhaps because we don't load Rails defaults in config, and also don't have
+  # config.active_record.belongs_to_required_by_default
+  # See also https://github.com/MushroomObserver/mushroom-observer/issues/1072
+  Enabled: false
+Rails/RedundantTravelBack: # new in 2.12
+  Enabled: true
+Rails/RenderInline: # new in 2.7
+  Enabled: true
```

Comment:
Rubocop seems to think that any inline rendering is a cardinal sin.
https://rails.rubystyle.guide/#inline-rendering
I don't understand why.

We render inline in 6 places. See below.
I'd be happy to template any or all of them if other developers would like that.
```
app/controllers/ajax_controller/exif.rb:38:5: C: Rails/RenderInline: Prefer using a template over inline rendering.
    render(inline: "<%= make_table(@data) %>")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/ajax_controller/export.rb:24:5: C: Rails/RenderInline: Prefer using a template over inline rendering.
    render(inline: "<%= image_exporter(@image.id, @image.ok_for_export) %>")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/ajax_controller/pivotal.rb:23:5: C: Rails/RenderInline: Prefer using a template over inline rendering.
    render(inline: "<%= pivotal_story(@story) %>")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/ajax_controller/pivotal.rb:29:5: C: Rails/RenderInline: Prefer using a template over inline rendering.
    render(inline: "<%= pivotal_vote_controls(@story) %>")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/ajax_controller/pivotal.rb:37:5: C: Rails/RenderInline: Prefer using a template over inline rendering.
    render(inline: "<%= pivotal_comment(@comment, @num) %>")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/ajax_controller/vote.rb:32:5: C: Rails/RenderInline: Prefer using a template over inline rendering.
    render(inline: %(<div> ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I can't imagine why adding a blank template here would serve any useful purpose.  Rendering these things in-line in the ajax controller seems like the perfect way to do it to me.  The whole point is that they are rending replacements for bits of HTML already within a template.